src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -4197,10 +4197,6 @@ void SmartScript::FillScript(SmartAIEventList e, WorldObject* obj, AreaTrigger c
         }
         mEvents.push_back((*i));//NOTE: 'world(0)' events still get processed in ANY instance mode
     }
-    if (mEvents.empty() && obj)
-        sLog->outErrorDb("SmartScript: Entry %u has events but no events added to list because of instance flags.", obj->GetEntry());
-    if (mEvents.empty() && at)
-        sLog->outErrorDb("SmartScript: AreaTrigger %u has events but no events added to list because of instance flags. NOTE: triggers can not handle any instance flags.", at->entry);
```

Comment:
why removing those?

Reply:
This was removed by trinitycore. The SAI is correct, the errro information was not to be shown on the console [1496](https://github.com/azerothcore/azerothcore-wotlk/issues/1496)

Reply:
@grimgravy thanks for the answer